flowetl/flowetl/flowetl/operators/staging/sql/append_sightings_to_main_table.sql

```diff
@@ -0,0 +1,12 @@
+BEGIN;
+
+-- Partitions are inclusive on lower, exclusive on upper
+ALTER TABLE reduced.sightings
+ATTACH PARTITION reduced.sightings_{{ds_nodash}} FOR VALUES FROM (date('{{ds_nodash}}')) TO (date('{{ds_nodash}}')+ 1) ;
+
+ALTER TABLE reduced.sightings_{{ds_nodash}}
+DROP CONSTRAINT sightings_{{ds_nodash}}_check;
```

Comment:
Why's the constraint being dropped out of curiosity?

Reply:
Constraint isn't necessary once the partition is attached (it's just duplicating what the declarative partitioning is doing). But it's required earlier to prevent postgres needing to do a full table scan when attaching the partition.